app/utils/updatePlan.server.ts

```diff
@@ -30,7 +30,7 @@ export const updatePlan = async ({
     }
 
     if (limit) {
-      options.input.enterpriseLimit = limit
+      options.input.monthlyUserLimit = limit
```

Comment:
This if statement should be removed for now since the monthly user limit is not yet fully supported. We will have to add some UI components to support allowing users to change this amount. For now I think we should remove the whole `if (limit)` block.